app/main.py

```diff
@@ -1,6 +1,26 @@
+import os
+
+import requests
+from typing import Dict
+
+API_KEY = os.environ["API_KEY"]
+FILTERING = "Paris"
```

Comment:
filtering is not that good name here

Reply:
In the beginning, I want to use CITY but...